kraken-std/src/kraken/std/dist.py

```diff
@@ -73,12 +73,7 @@ def execute(self) -> None:
         output_file.parent.mkdir(parents=True, exist_ok=True)
         with wopen_archive(output_file, archive_type) as archive:
             for resource in self.resources.get():
-                if resource.options.arcname is not None:
-                    arcname = resource.options.arcname
-                elif isinstance(resource, BinaryArtifact):
```

Comment:
this is where the bug was, at this point `resource` may no longer be a `BinaryArtifact`, it is a `ConfiguredResource` (as the type of the property attest)